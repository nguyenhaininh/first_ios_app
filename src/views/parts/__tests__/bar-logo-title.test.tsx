/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import BarLogoTitle from '../bar-logo-title';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <BarLogoTitle >
      hogehoge
    </BarLogoTitle>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
