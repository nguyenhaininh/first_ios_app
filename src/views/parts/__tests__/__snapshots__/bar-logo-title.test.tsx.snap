// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#ffcd00",
        "height": 20,
        "width": 20,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#2f3f62",
        "fontSize": 16,
        "fontWeight": "bold",
        "marginLeft": 11,
      }
    }
  >
    hogehoge
  </Text>
</View>
`;
